tests/cdn/ck/createCKCdnBaseBundlePack.test.ts

```diff
@@ -76,4 +87,33 @@ describe( 'createCKCdnBaseBundlePack', () => {
 			]
 		} );
 	} );
+
+	it( 'should throw an error if the requested version is different than the installed one', async () => {
+		await loadCKEditor( '43.0.0' );
+		await expect( async () => loadCKEditor( '42.0.0' ) ).rejects.toThrowError(
+			'CKEditor 5 version 43.0.0 is already loaded from CDN. Requested version is 42.0.0.'
```

Comment:
I just noticed that my changes would require changes in tests. Sorry!
```suggestion
			'CKEditor 5 version 43.0.0 is already loaded from CDN. The requested version is 42.0.0.'
```